Audio/BnkCompiler/Compiler.cs

```diff
@@ -16,24 +19,282 @@ public class CompileResult
         public CompilerData Project { get; set; }
         public PackFile OutputBnkFile { get; set; }
         public PackFile OutputDatFile { get; set; }
-        public PackFile NameList { get; set; }
+        public PackFile OutputStatesDatFile{ get; set; }
     }
 
     internal static class CompilerConstants
     {
         public static readonly string Game_Warhammer3 = "Warhammer3";
-        public static readonly string Language_English = "English(uk)";
-        public static readonly string Mixers_Diplomacy = "54848735";
+        public static readonly Dictionary<string, uint> VanillaIds = new Dictionary<string, uint>()
+        {
+            // Ids for Event mixers
```

Comment:
Large lists of constants does not really belong alongside business logic.
I suggest moving it to a different class which implements an interface.The class should be inside the warhammer 3 specific folder.

Reply:
Cool, will do that